tensorflow/lite/experimental/micro/examples/mnist_demo/model/mnist_train_and_export_to_tfl_micro.py

```diff
@@ -0,0 +1,312 @@
+# Copyright 2018 The TensorFlow Authors. All Rights Reserved.
```

Comment:
Can you put python training and conversion code into a python notebook, and link in this repo?  That is the strategy we use for other examples (see examples/hello_world/create_sine_model.ipynb for an example).

Reply:
Hi @njeffrie I've just added a notebook for this now.